docs/book/src/development/development.md

```diff
@@ -42,3 +42,103 @@ After generating `infrastructure-components.yaml`, replace the `us.gcr.io/k8s-ar
+              OPENSTACK_CLOUD_CACERT_B64=$(base64 -w /path/to/mycloud-ca.crt)
+```
+
+CAPO deployed in the local kind cluster will automatically pick up a `cacert` defined in your `clouds.yaml` so you will see servers created in OpenStack without specifying `OPENSTACK_CLOUD_CACERT_B64`. However, the cacert won't be deployed to those servers, so kubelet will fail to start.
+
+### Support for clouds with multiple external networks
+
+If your cloud contains only a single external network CAPO will automatically select that network for use by a deployed cluster. However, if there are multiple external networks CAPO will log an error and fail to create any machines. In this case you must pass the id of an external network to use explicitly with `OPENSTACK_EXTERNAL_NETWORK_ID`, i.e.:
+
+```bash
+make test-e2e OPENSTACK_CLOUD_YAML_FILE=/path/to/clouds.yaml OPENSTACK_CLOUD=my_cloud \
+              OPENSTACK_EXTERNAL_NETWORK_ID=27635f93-583d-454e-9c6d-3d305e7f8a22
+```
+
+`OPENSTACK_EXTERNAL_NETWORK_ID` must be specified as a uuid. Specifying by name is not supported.
+
+You can list available external networks with:
+
+```bash
+$ openstack network list --external
++--------------------------------------+----------+--------------------------------------+
+| ID                                   | Name     | Subnets                              |
++--------------------------------------+----------+--------------------------------------+
+| 27635f93-583d-454e-9c6d-3d305e7f8a22 | external | be64cd07-f8b7-4705-8446-26b19eab3914 |
+| cf2e83dc-545d-490f-9f9c-4e90927546f2 | hostonly | ec95befe-72f4-4af6-a263-2aec081f47d3 |
++--------------------------------------+----------+--------------------------------------+
+```
+
+### OpenStack prerequisites
+
+The file `test/e2e/data/e2e_conf.yaml` and the test templates under `test/e2e/data/infrastructure-openstack` reference several OpenStack resources which must exist before running the test:
+
+* Glance images
+  * `cirros-0.5.1-x86_64-disk`
+    * Download from https://docs.openstack.org/image-guide/obtain-images.html
+  * `ubuntu-2004-kube-v1.18.15`
+    * Download from https://storage.googleapis.com/artifacts.k8s-staging-capi-openstack.appspot.com/test/ubuntu/2021-03-27/ubuntu-2004-kube-v1.18.15.qcow2
+    * Or generate using the `images/capi` directory from https://github.com/kubernetes-sigs/image-builder
+* Flavors
+  * `m1.medium`: used by control plane
+  * `m1.small`: used by workers
+  * `m1.tiny`: used by bastion
+
+## Running E2E tests using rootless podman
+
+You can use unprivileged podman to:
+* Build the CAPO image
+* Deploy the kind cluster
+
+To do this you need to configure the host appropriately and pass `PODMAN=1` to make, i.e.:
+
+```bash
+make test-e2e OPENSTACK_CLOUD_YAML_FILE=/path/to/clouds.yaml OPENSTACK_CLOUD=my_cloud \
+              PODMAN=1
+```
+
+### Host configuration
+
+Firstly, you must be using kernel >=5.11. Fedora 34 is the oldest distribution with this kernel.
```

Comment:
oldest ? latest?

Reply:
oldest is correct here. It's also the latest! However, this statement will still be correct when it is no longer the oldest.